README.md

```diff
@@ -1,66 +1,38 @@
 # freezeyt
-Generator of static web pages created by the Czech community of PyLadies.
-
-
-## Table of contents
-* [Project goal](#Project-goal)
-* [What is the project about?](#What-is-the-project-about?)
-    * [What is NOT the project about?](#What-is-NOT-the-project-about?)
-* [Installation](#Installation)
-* [Usage](#Usage)
-* [Contributing](#Contributing)
-    * [How to start contribute](#How-to-contribute)
-* [Authors](#Authors)
-* [Licence](#Licence)
-    
+Generator of static web pages created by the Czech community of python programming language.
 
 
 ## Project goal
 Main goal of this project is to create a versatile freezer to generate static web pages from web application. The freezer aims to be compatible with any Python Web framework. 
 
 
 ## What IS the project about?
-The idea is to create a new freezer in order to generate static web pages. It should also help PyLadies who've already taken their first steps with the Python programming language and now they are looking for some real projects where they could not only put their programming skills to the test but also to improve them.
-
-The Czech Python community uses a lot of static web pages that are generated from a web application for community purposes e.g. workshops, courses, or meetups.
-
-The community has been so far relying on the following projects in order to generate the static web content:
+The idea is to create a new freezer in order to generate static web pages. It should also help the czech beginner pythonistas who've already taken their first steps with the Python programming language and now they are looking for some real projects where they could not only put their programming skills to the test but also to improve them.
 
-* [flask-frozen](https://pythonhosted.org/Frozen-Flask/)
-* community module [elsa](https://github.com/pyvec/elsa/)
 
-
-
-The new freezer ought to be based on the old elsa one, but such that it can be used with an arbitrary Python Web application framework, e.g.:
-
-* [Django](https://www.djangoproject.com/)
-* [Tornado](https://www.tornadoweb.org/en/stable/)
-* etc.
  
 ### What the project IS NOT about?
 I have no idea so far.
 
+
 ## Installation
-It is higly recommended to create a separate virtual enviroment for this project.
 
-For testing the project it's necessary to install pytest:
-```
-$ python -m pip install pytest
-```
+There are no other installation requirements for this project.
```

Comment:
Actually, there are, they are written in `requirements.txt`

Reply:
And I think you can use this expresion to install all requirements in requirements file:

`python -m pip install requirements.txt`

Reply:
Isn't it `python -m pip install -r requirements.txt`?

Reply:
ja vcerejsi stream ještě nedokoukal myslel jsem že už se petr k readme dale nevyjadřoval :)